spec/models/seed_spec.rb

```diff
@@ -152,4 +152,32 @@
     end
   end
 
+  context 'photos' do
+    before(:each) do
+      @seed = FactoryGirl.create(:seed)
+      @photo = FactoryGirl.create(:photo)
```

Comment:
No idea. I'm really in "copy and paste what someone else did, then poke it a little" mode when it comes to the spec files.